http://hg.secdev.org/scapy/rev/443b35c8c6de

Fixed startup config file reading problem when scapy is used as a library

--- scapy.py.orig	2008-01-30 19:47:55.000000000 +0300
+++ scapy.py	2008-01-30 19:48:42.000000000 +0300
@@ -12190,4 +12190,5 @@
 if __name__ == "__main__":
     interact()
 else:
-    read_config_file(DEFAULT_CONFIG_FILE)
+    if DEFAULT_CONFIG_FILE:
+        read_config_file(DEFAULT_CONFIG_FILE)
